packages/opencensus-core/src/metrics/utils.ts

```diff
@@ -40,3 +41,36 @@ export function hashLabelValues(labelValues: LabelValue[]): string {
 export function initializeDefaultLabels(count: number): LabelValue[] {
   return new Array(count).fill(UNSET_LABEL_VALUE);
 }
+
+// Checks if the specified collection is a LengthAttributeInterface.
+// tslint:disable-next-line:no-any
+export function isLengthAttributeInterface(obj: any):
```

Comment:
Can you use `unknown` here instead of `any`? (Same for other functions below)

Reply:
Good Point, `unknown` type is available since TypeScript 3.0, we are still using [2.9.0 version](https://github.com/census-instrumentation/opencensus-node/issues/279) for core package. For now, I have added TODO comment: 229a9b7afe078f27c6896607aa64f09cfd0acc09. PTAL once you have time.

Reply:
The TODO comment looks great, but I'd suggest just putting it right above this function and then removing the commented out code below it. LGTM to me overall though so approving again!